components/patient-access-rules/ui/src/main/resources/PhenoTips/AccessRightsManagement.xml

```diff
@@ -1075,7 +1076,7 @@
       var row = new Element('tr', {'class' : (highlight === true ? 'new' : '')});
       row.insert(new Element('td').insert(new Element('span', {'class' : 'fa fa-' + c.type}).update(' ')));
       row.insert(new Element('td')
-              .insert(c.name)
+              .insert(c.name ? c.name : XWiki.Model.resolve(c.id, XWiki.EntityType.DOCUMENT).name)
```

Comment:
do we need the same code (as on line 1056 above) here as well, wouldn't that fix fix this too?

Reply:
Line 1056 is for permissions detains on top of patioent form, line 1079 is for collaborators suggestions selected in the permissions dialog.
The resolving logic is adopted from studying XWiki code, userpicker.js in particular. I assumed, it's a right approach if its in the platform codebase.

Reply:
I guess it is then. I was jusf surprised that is done on the front-end and not om the back-end before even sending data to the front-end